cli/src/ui.rs

```diff
@@ -57,6 +57,11 @@ impl Write for &BuiltinPager {
     }
 
     fn write(&mut self, buf: &[u8]) -> io::Result<usize> {
+        // Since minus::Pager holds the rx end internally, push_str() doesn't
+        // fail even after the paging thread gets terminated.
+        if self.dynamic_pager_thread.is_finished() {
+            return Err(io::Error::new(io::ErrorKind::BrokenPipe, "pager quit"));
```

Comment:
suggestion: Unless you want the error message here (I don't think it's used at any point?), you can avoid an allocation for the error message by using `io::Error::from(io::ErrorKind::BrokenPipe)`.

Reply:
Allocation isn't important, but I like it. Switched to from/into.